js/src/lib/Details/Markdown.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import marked from 'marked';
 import xss from 'xss';
+import hljs from 'highlight.js';
```

Comment:
How much does this increase the bundle?

Reply:
Where can I check this?

Reply:
highlight.js is 20K without minification. Probably it will add +10K at least to the current website.

Reply:
https://deploy-preview-716--yarnpkg.netlify.com/en/

Reply:
I’m wondering if there could be a way to have this only in the bundle where needed 🤔

Reply:
I think you're loading Highlight.js twice now - Once in `_layouts/default.html` and once here.

I also think that `https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js` contains every language supported by Highlight.js. I went to the Highlight.js site (https://highlightjs.org/download/) and downloaded a build that only supports Markdown, and it's only 9.4 KB minified / 4.8 KB minified and gzipped. That might be the best thing to do here? I wonder if you could do that using the npm package. Otherwise, I think downloading a minimal Highlight.js and sticking it in the repo is fine.

Here's webpack's docs on "code splitting": https://webpack.js.org/guides/code-splitting/. It also talks about dynamic imports.

Reply:
I think we can start with supporting JS and CSS for now?